src/Tmds.Ssh/Ed25519PrivateKey.cs

```diff
@@ -0,0 +1,66 @@
+// This file is part of Tmds.Ssh which is released under MIT.
+// See file LICENSE for full license details.
+
+using System;
+using System.Buffers;
+using Org.BouncyCastle.Math.EC.Rfc8032;
+
+namespace Tmds.Ssh;
+
+sealed class Ed25519PrivateKey : PrivateKey
+{
+    private const int _privateKeySize = 32;
+
+    // Contains the private and public key as one block of bytes from the
+    // serialized OpenSSH key data.
+    private readonly byte[] _keyData;
```

Comment:
I used a single `byte[]` as that's how it is encoded in the raw OpenSSH key and the BouncyCastle API accepts an offset for the private and public key. Happy to just split that out in the parser and store it as 2 separate arrays if you prefer that.

Reply:
Yes, please split into separate arrays for the private and public key.